code/modules/events/money_hacker.dm

```diff
@@ -38,15 +38,14 @@
 		message = "The hack attempt has succeeded."
 
 		//subtract the money
-		var/amount = affected_account.money * 0.8 + (rand(2,4) - 2) / 10
+		var/amount = min(affected_account.money * 0.8 + rand(1,50), affected_account.money)
 
 		//create a taunting log entry
-		var/name = pick("","[pick("Biesel","New Gibson")] GalaxyNet Terminal #[rand(111,999)]","your mums place","nantrasen high CommanD")
+		var/name = pick("","[pick("Biesel","New Gibson")] ATM Terminal #[rand(111,999)]","your mums place","ca0s !nsurg3ncy", "nigeran prince")
```

Comment:
???????????

Reply:
They're fake names for the transaction, they're supposed to be goofy.

Reply:
replace nigerian prince

Reply:
gotya